docs/getting-started/server/portal.md

```diff
@@ -1,12 +1,61 @@
-# Bitwarden Portal
+# System Management Portal
+
+<bitwarden>
+
+:::info Portal naming
+
+This documentation refers to the deployment of the `Admin` application in our `server` repository.
+To disambiguate this application from others in the Bitwarden landscape, we refer to it as follows:
+
+- For **Cloud-Hosted** Instances (internal to Bitwarden) &rarr; **Bitwarden Portal**
+- For **Self-Hosted** Instances &rarr; **System Management Portal**
+
+:::
+
+</bitwarden>
```

Comment:
This is an important (and potentially confusing) naming convention, I tend to think it should be shown to everyone (i.e. not wrapped in `<bitwarden>` tags).

Reply:
Good idea.  I've removed the tags.